atspi-common/src/lib.rs

```diff
@@ -209,7 +209,7 @@ pub enum ScrollType {
 /// The argument in the `Announcement` event is named `politeness`.
 #[derive(Clone, Copy, Debug, Default, PartialEq, Eq, Hash, Serialize, Deserialize, Type)]
 #[repr(i32)]
-pub enum Live {
+pub enum Politeness {
```

Comment:
Big thanks for this. 
Maybe in a certain context `Live` made sense at some point. (I suppose 'the way we may interrupt 'live' (ongoing) speech'.)
I think `Politeness` is much more descriptive.